doto-frontend/src/components/pages/Calendar/CalendarComponent.js

```diff
@@ -0,0 +1,112 @@
+import React from "react";
+import "./Calendar.css";
+import Paper from "@material-ui/core/Paper";
+import { MuiThemeProvider, createMuiTheme } from "@material-ui/core/styles";
+import { ViewState, EditingState, IntegratedEditing } from "@devexpress/dx-react-scheduler";
+import {
+    Scheduler,
+    WeekView,
+    MonthView,
+    DayView,
+    ViewSwitcher,
+    Toolbar,
+    DateNavigator,
+    TodayButton,
+    Appointments,
+    AppointmentForm,
+    ConfirmationDialog,
+    EditRecurrenceMenu,
+    AllDayPanel,
+    AppointmentTooltip,
+} from "@devexpress/dx-react-scheduler-material-ui";
+import { green } from "@material-ui/core/colors";
+const theme = createMuiTheme({ palette: { type: "light", primary: green } });
+
+let today = new Date();
+
+// Calculating the current date
+function getDate() {
+    const day = String(today.getDate()).padStart(2, "0");
+    const month = String(today.getMonth() + 1).padStart(2, "0");
+    const year = today.getFullYear();
+    today = year + "-" + month + "-" + day; // yyyy-mm-ddT24.00.000
```

Comment:
This matches the format required by the back-end team correct?

Reply:
This is just calculating the current date which is used for setting the default current date in the calendar on line 86

Reply:
Okay. Just making sure.